src/pages/RegistrationPage/RegistrationPage.jsx

```diff
@@ -16,68 +10,6 @@ const RegistrationPage = () => {
         </div>
         <div className={css.registrationFormContainer}>
           <RegisterForm />
-         
-          {/* <div className={css.registrationBox}>
-            <div className={css.logo}>
-              <img src={walletLogo} alt="Wallet logo" />
-            </div>
-            <ul className={css.inputList}>
-              <li className={css.inputBox}>
-                <div className={css.inputBox__icon}>
-                  <img src={email} alt="Email icon" />
-                </div>
-                <input
-                  type="email"
-                  id="email"
-                  placeholder="E-mail"
-                  className={css.inputBox__input}
-                />
-              </li>
-              <li className={css.inputBox}>
-                <div className={css.inputBox__icon}>
-                  <img src={lock} alt="Lock icon" />
-                </div>
-                <input
-                  type="password"
-                  id="password"
-                  placeholder="Password"
-                  className={css.inputBox__input}
-                />
-              </li>
-              <li className={css.inputBox}>
-                <div className={css.inputBox__icon}>
-                  <img src={lock} alt="Lock icon" />
-                </div>
-                <input
-                  type="password"
-                  id="confirmPassword"
-                  placeholder="Confirm password"
-                  className={css.inputBox__input}
-                />
-              </li>
-              <li className={css.inputBox}>
-                <div className={css.inputBox__icon}>
-                  <img src={userIcon} alt="User icon" />
-                </div>
-                <input
-                  type="text"
-                  id="firstName"
-                  placeholder="First name"
-                  className={css.inputBox__input}
-                />
-              </li>
-            </ul>
-            <ul className={css.registrationBtnList}>
-              <li>
-                <button className={css.registrationBtnCurrentPage}>
-                  REGISTER
-                </button>
-              </li>
-              <li>
-                <button className={css.registrationBtn}>LOG IN</button>
-              </li>
-            </ul>
-          </div> */}
```

Comment:
a to nie jest jakiś aktywny comment?
ten usunięty?

Reply:
nie nie, to jest kod któy już nie był używany. Teraz całość znajduje się RegistrationForm. Także to na spokojnie można usunąć